presto-spi/src/main/java/com/facebook/presto/spi/CatalogSchemaTableName.java

```diff
@@ -14,19 +14,16 @@
 
 package com.facebook.presto.spi;
 
-import com.fasterxml.jackson.annotation.JsonCreator;
-import com.fasterxml.jackson.annotation.JsonValue;
-
 import java.util.Objects;
 
 import static com.facebook.presto.spi.SchemaUtil.checkNotEmpty;
+import static java.util.Objects.requireNonNull;
 
-public class CatalogSchemaTableName
+public final class CatalogSchemaTableName
 {
     private final String catalogName;
     private final SchemaTableName schemaTableName;
 
-    @JsonCreator
     public static CatalogSchemaTableName valueOf(String catalogSchemaTableName)
```

Comment:
Let's remove this method entirely, since it's not used and doesn't handle quoting properly.